### ISSUE TITLE
It seems I cannot customize the bottom statusline

### ISSUE THREAD

**Issue by LuxGiammi** [wsdjeg/SpaceVim#3901]
## Expected behavior, english is required
I cannot customize the bottom statusline from the `init.toml` file.
It seems like the configuration is completely ignored.

Note: it seems a problem similar to #3835 but it also affects the information displayed on the statusline
Note2: I have Hack-Nerd-Font installed (I think it's visible on the attached screenshot)

## The reproduce ways from Vim starting (Required!)
1. Open init.toml
2. Paste this in:
```
statusline_separator = "arrow"
statusline_inactive_separator = "arrow"
statusline_left_sections = [
  'winnr',
  'major mode',
  'filename',
  'fileformat',
  'minor mode lighters',
  'version control info',
  'search status'
]
# right sections of statusline
statusline_right_sections = [
  'cursorpos',
  'percentage',
  'input method',
  'date',
  'time'
]
```
3. Save and close
4. Open up SpaceVim (`vim -u ~/.SpaceVim/vimrc`)
5. The behavior is the one highlighted in the following screenshot:
![1602769637](https://user-images.githubusercontent.com/17674050/96138182-ca640780-0efd-11eb-8eb5-9ea43a6a057a.png)

Notice that the upper bar has the separators specified in the configuration, while the bottom one not only has no separator at all, but also misses a lot of the specified information (such as date, time or the major mode). If I press 'e' to open up a new file, the bottom bar changes, but the information displayed on it does not reflect the requested information from the config file. I can enable some of them by using SPC-t-m, but some others do not work at all (for instance SPC-t-m-M does not show the major mode).

## Debug info

Please press <kbd>SPC h I</kbd>, debug info will be put into clipboard, then paste all content below.

## Environment Information

- OS: linux
- vim version: 8.2.1844
- neovim version: -
- SpaceVim version: 1.6.0-dev
- SpaceVim status: bd40c093

```
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
```

## Output of the `:SPDebugInfo!`

<details><summary> SpaceVim debug information </summary>

### SpaceVim options :

```toml
  auto_disable_touchpad = 1
  autocomplete_method = 'asyncomplete'
  autocomplete_parens = 1
  buffer_index_type = 4
  checkinstall = 1
  colorscheme = 'gruvbox'
  colorscheme_bg = 'dark'
  colorscheme_default = 'desert'
  commandline_prompt = '➭'
  custom_color_palette = []
  custom_plugins = []
  data_dir = '/home/luxgiammi/.cache/'
  debug_level = 1
  default_indent = 2
  disabled_plugins = []
  enable_ale = 0
  enable_bepo_layout = 0
  enable_cursorcolumn = 0
  enable_cursorline = 1
  enable_debug = 0
  enable_googlesuggest = 0
  enable_guicolors = 1
  enable_insert_leader = 1
  enable_javacomplete2_py = 0
  enable_key_frequency = 0
  enable_language_specific_leader = 1
  enable_neocomplcache = 0
  enable_neomake = 1
  enable_os_fileformat_icon = 0
  enable_powerline_fonts = 1
  enable_projects_cache = 1
  enable_statusline_bfpath = 0
  enable_statusline_mode = 0
  enable_statusline_tag = 1
  enable_tabline_filetype_icon = 1
  enable_tabline_ft_icon = 0
  enable_vimfiler_filetypeicon = 0
  enable_vimfiler_gitstatus = 0
  enable_vimfiler_welcome = 1
  enable_ycm = 0
  error_symbol = '✖'
  escape_key_binding = 'jk'
  expand_tab = 1
  filemanager = 'vimfiler'
  filetree_direction = 'right'
  filetype_icons = {}
  force_global_config = 0
  gitcommit_issue_icon = ''
  gitcommit_pr_icon = ''
  github_username = ''
  guifont = ''
  hiddenfileinfo = 1
  home_files_number = 6
  hosts_url = 'https://raw.githubusercontent.com/racaljk/hosts/master/hosts'
  info_symbol = 'ⓘ'
  keep_server_alive = 1
  language = ''
  leader_guide_default_group_name = ''
  leader_guide_flatten = 1
  leader_guide_hspace = 5
  leader_guide_max_size = 0
  leader_guide_position = 'botright'
  leader_guide_run_map_on_popup = 1
  leader_guide_sort_horizontal = 0
  leader_guide_submode_mappings = {'<C-C>': 'win_close'}
  leader_guide_vertical = 0
  lint_on_save = 1
  lint_on_the_fly = 0
  max_column = 120
  plugin_bundle_dir = '/home/luxgiammi/.cache/vimfiles/'
  plugin_manager = 'dein'
  plugin_manager_processes = 16
  plugin_name = 'vim-grepper'
  project_rooter_automatically = 1
  project_rooter_outermost = 1
  project_rooter_patterns = ['.git/', '_darcs/', '.hg/', '.bzr/', '.svn/', '.SpaceVim.d/']
  projects_cache_num = 20
  realtime_leader_guide = 1
  relativenumber = 1
  search_tools = ['rg', 'ag', 'pt', 'ack', 'grep', 'findstr', 'git']
  sidebar_direction = ''
  sidebar_width = 30
  smartcloseignoreft = ['tagbar', 'vimfiler', 'defx', 'SpaceVimRunner', 'SpaceVimREPL', 'SpaceVimQuickFix', 'HelpDescribe', 'VebuggerShell', 'VebuggerTerminal', 'SpaceVimTabsManager']
  smartcloseignorewin = ['__Tagbar__', 'vimfiler:default']
  snippet_engine = 'neosnippet'
  src_root = 'E:\sources\'
  statusline_inactive_separator = 'arrow'
  statusline_iseparator = 'nil'
  statusline_left_sections = ['winnr', 'major mode', 'filename', 'fileformat', 'minor mode lighters', 'version control info', 'search status']
  statusline_right_sections = ['cursorpos', 'percentage', 'input method', 'date', 'time']
  statusline_separator = 'arrow'
  statusline_unicode_symbols = 1
  terminal_cursor_shape = 2
  todo_labels = ['@fixme', '@question', '@todo', '@idea']
  update_retry_cnt = 3
  version = '1.6.0-dev'
  vim_help_language = 'en'
  vimcompatible = 1
  warning_symbol = '⚠'
  wildignore = '*/tmp/*,*.so,*.swp,*.zip,*.class,tags,*.jpg,*.ttf,*.TTF,*.png,*/target/*,.git,.svn,.hg,.DS_Store,*.svg'
  windows_index_type = 3
  windows_leader = ''
  windows_smartclose = ''
```

### SpaceVim layers :

```toml
[[layers]]
  name="autocomplete"
  auto_completion_return_key_behavior='complete'
  auto_completion_tab_key_behavior='smart'
[[layers]]
  name="checkers"
[[layers]]
  name="format"
[[layers]]
  name="edit"
[[layers]]
  name="ui"
[[layers]]
  name="core"
[[layers]]
  name="core#banner"
[[layers]]
  name="core#statusline"
[[layers]]
  name="core#tabline"
[[layers]]
  name="shell"
  default_position='top'
  default_height=30
```

### SpaceVim Health checking :

SpaceVim clipboard support check report:
Checking +clipboard:
      SUCCEED!
SpaceVim environment check report:
Current progpath: vim(/usr/local/bin/vim)
version: 802
OS: linux
[shell, shellcmdflag, shellslash]: ['/bin/bash', '-c', 0]
SpaceVim lua support check report:
Checking +lua:
      Failed : to support +lua, you need recompile your vim with +lua support.
SpaceVim python support check report:
Checking +python3:
      SUCCEED!
Checking +python:
      Failed : to support +python, Please install vim-gtk, or build from sources.

### SpaceVim runtime log :

```log
[ SpaceVim ] : logger file  does not exists, only log for current process will be shown!
[ SpaceVim ] [15:47:11] [ Info ] Startup with no argv, current dir is used: ~
[ SpaceVim ] [15:47:11] [ Info ] find local conf: /home/luxgiammi/.SpaceVim.d/init.toml
[ SpaceVim ] [15:47:11] [ Info ] loading cached local conf: /home/luxgiammi/.cache//SpaceVim/conf/_home_luxgiammi__SpaceVim_d_init_toml.json
[ SpaceVim ] [15:47:11] [ Info ] start to apply config [local]
[ SpaceVim ] [15:47:11] [ Info ] SpaceVim server startup at:/tmp/spacevim_vim_server
[   rooter ] [15:47:11] [ Info ] Load projects cache from: /home/luxgiammi/.cache/SpaceVim/projects.json
[   rooter ] [15:47:11] [ Info ] Start to find root for: /home/luxgiammi/
[   rooter ] [15:47:11] [ Info ]         (.SpaceVim.d/):/home/luxgiammi/
[   rooter ] [15:47:11] [ Info ] buffer name: No Name
[   rooter ] [15:47:11] [ Info ] same as current directory, no need to change.
[ SpaceVim ] [15:47:11] [ Info ] try to open SpaceVim welcome page
[   rooter ] [15:47:11] [ Info ] buffer name: No Name
[   rooter ] [15:47:11] [ Info ] same as current directory, no need to change.
[   rooter ] [15:50:03] [ Info ] Start to find root for: /home/luxgiammi/
[   rooter ] [15:50:03] [ Info ]         (.SpaceVim.d/):/home/luxgiammi/
[   rooter ] [15:50:03] [ Info ] buffer name: No Name
[   rooter ] [15:50:03] [ Info ] same as current directory, no need to change.
[ SpaceVim ] [15:50:03] [ Info ] key binding guide float statusline winid:1004
[ SpaceVim ] [15:50:03] [ Info ] close float statusline winid:1004
[   rooter ] [15:50:03] [ Info ] Start to find root for: /home/luxgiammi/
[   rooter ] [15:50:03] [ Info ]         (.SpaceVim.d/):/home/luxgiammi/
[   rooter ] [15:50:03] [ Info ] buffer name: No Name
[   rooter ] [15:50:03] [ Info ] same as current directory, no need to change.
[ SpaceVim ] [15:50:03] [ Info ] key binding guide float statusline winid:1006
[ SpaceVim ] [15:50:03] [ Info ] close float statusline winid:1006
[   rooter ] [15:50:03] [ Info ] Start to find root for: /home/luxgiammi/
[   rooter ] [15:50:03] [ Info ]         (.SpaceVim.d/):/home/luxgiammi/
[   rooter ] [15:50:03] [ Info ] buffer name: No Name
[   rooter ] [15:50:03] [ Info ] same as current directory, no need to change.
[ SpaceVim ] [15:50:03] [ Info ] key binding guide float statusline winid:1008
[ SpaceVim ] [15:50:04] [ Info ] close float statusline winid:1008
[   rooter ] [15:50:05] [ Info ] Start to find root for: /home/luxgiammi/
[   rooter ] [15:50:05] [ Info ]         (.SpaceVim.d/):/home/luxgiammi/
[   rooter ] [15:50:05] [ Info ] buffer name: No Name
[   rooter ] [15:50:05] [ Info ] same as current directory, no need to change.
[   rooter ] [15:51:50] [ Info ] Start to find root for: /home/luxgiammi/SpaceVimLayers
[   rooter ] [15:51:50] [ Info ]         (.SpaceVim.d/):/home/luxgiammi/
[   rooter ] [15:51:50] [ Info ] buffer name: SpaceVimLayers
[   rooter ] [15:51:50] [ Info ] same as current directory, no need to change.
[   rooter ] [15:52:04] [ Info ] Start to find root for: /home/luxgiammi/
[   rooter ] [15:52:04] [ Info ]         (.SpaceVim.d/):/home/luxgiammi/
[   rooter ] [15:52:04] [ Info ] buffer name: No Name
[   rooter ] [15:52:04] [ Info ] same as current directory, no need to change.
[ SpaceVim ] [15:52:04] [ Info ] key binding guide float statusline winid:1011
[ SpaceVim ] [15:52:05] [ Info ] close float statusline winid:1011
[   rooter ] [15:52:05] [ Info ] Start to find root for: /home/luxgiammi/
[   rooter ] [15:52:05] [ Info ]         (.SpaceVim.d/):/home/luxgiammi/
[   rooter ] [15:52:05] [ Info ] buffer name: No Name
[   rooter ] [15:52:05] [ Info ] same as current directory, no need to change.
[ SpaceVim ] [15:52:05] [ Info ] key binding guide float statusline winid:1013
[ SpaceVim ] [15:52:11] [ Info ] close float statusline winid:1013
[   rooter ] [15:52:11] [ Info ] Start to find root for: /tmp/vomd1Sg/2/issue_report.md
[   rooter ] [15:52:11] [ Info ] buffer name: /tmp/vomd1Sg/2/issue_report.md
[   rooter ] [15:52:11] [ Info ] same as current directory, no need to change.
```
</details>

**Comment by wsdjeg** [comment]
ohh, sorry, I just have found the type in doc, I will fix it.

**Comment by LuxGiammi** [comment]
I did `cd .SpaceVim && git pull` and changed `statusline_inactive_separator = "arrow"` to `statusline_iseparator = "arrow"`, but the problems persist (both the separator and information ones). Anyway, thanks a lot for your quick feedback. :smiley:

**Comment by LuxGiammi** [comment]
Update: @wsdjeg I applied the fix specified in the issue #3835 and if I change both `statusline_separator` and `statusline_iseparator` at least the separators work for the bottom bar too. If they differ, they fallback to 'arrow', I don't know why.

Other information for the statusbar specified in the `init.toml` file is missing.
